src/StoragelessSession/Http/SessionMiddleware.php

```diff
@@ -85,7 +94,8 @@ public function __construct(
         string $verificationKey,
         SetCookie $defaultCookie,
         Parser $tokenParser,
-        int $expirationTime
+        int $expirationTime,
+        int $refreshTime = self::DEFAULT_REFRESH_TIME
```

Comment:
Not really happy with this one - maybe make it mandatory, then set the default only when using the "helper constructors"?